beets/util/__init__.py

```diff
@@ -894,7 +907,7 @@ def command_output(
     if proc.returncode:
         raise subprocess.CalledProcessError(
             returncode=proc.returncode,
-            cmd=" ".join(map(str, cmd)),
+            cmd=" ".join(converted_cmd),
```

Comment:
I'm wondering where this `map(...)` came from historically... good thing to clean this up!

Reply:
Potentially lack of clarity regarding the output type of `convert_command_args`?

```python
    converted_cmd = convert_command_args(cmd)
```